pkg/storage/postgres/postgres.go

```diff
@@ -158,7 +158,7 @@ func (p *Postgres) ReadPage(ctx context.Context, store string, tupleKey *openfga
 	return iter.ToArray(opts)
 }
 
-func (p *Postgres) read(ctx context.Context, store string, tupleKey *openfgav1.TupleKey, opts *storage.PaginationOptions) (*sqlcommon.SQLTupleIterator, error) {
+func (p *Postgres) read(ctx context.Context, store string, tupleKey *openfgav1.TupleKey, opts *storage.PaginationOptions, sortByUlid bool) (*sqlcommon.SQLTupleIterator, error) {
```

Comment:
`sortByUlid` belongs as an option to the method. 5 function parameters is starting to get a bit wieldy.